specs/git/rebase_master_into_current_branch.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Rebase master into feature branch
+command: |-
+    git checkout
```

Comment:
nit: does this first command no-op? should we remove it?

Reply:
It's necessary for `git checkout -` to work properly later in the command